src/components/backdrop/backdrop.scss

```diff
@@ -23,6 +23,9 @@ md-backdrop {
   &._md-dialog-backdrop {
     z-index: $z-index-dialog - 1;
   }
+  &._md-panel-backdrop {
+    z-index: $z-index-dialog - 1;
```

Comment:
We can't assume this is the z-index. (A tooltip within a dialog is an example.) Yes, the default is $z-index-dialog, but we allow the user to customize that to any value they want. You could set this in the panel instead where you have access to the panel z-index.